src/bundle/lookup.js

```diff
@@ -0,0 +1,42 @@
+define([
+	"../core/likely_subtags",
+	"../core/remove_likely_subtags",
+	"../core/subtags",
+	"../util/array/for_each"
+], function( coreLikelySubtags, coreRemoveLikelySubtags, coreSubtags, arrayForEach ) {
+
+	/**
+	 * bundleLookup( minLanguageId )
+	 *
+	 * @Cldr [Cldr class]
+	 *
+	 * @cldr [Cldr instance]
+	 *
+	 * @minLanguageId [String] requested languageId after applied remove likely subtags.
+	 *
+	 * Ref: http://goo.gl/qpwZ4n
+	 */
+	return function( Cldr, cldr, minLanguageId ) {
+		var availableBundleMap = Cldr._availableBundleMap,
+			availableBundleMapQueue = Cldr._availableBundleMapQueue;
+
+		if ( availableBundleMapQueue.length ) {
+			arrayForEach( availableBundleMapQueue, function( bundle ) {
+				var existing, maxBundle, minBundle, subtags;
```

Comment:
Thanks. But, I read the below better.

Note, some variables depend on others being initialized. `subtags` is the only one I would inline in the `var` statement.